kubeflow/pipelines
# feat(sdk): rename CLI methods to 'create'

**Description of your changes:**

Aliases some CLI verbs to be more consistent.

To test on the `kfp` CLI:
`kfp run submit --help` should work, but raise a warning
`kfp run create --help` should work, but _not_ raise a warning
`kfp pipeline upload --help` should work, but raise a warning
`kfp pipeline create --help` should work, but _not_ raise a warning

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more 
about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).
